libs/cart/src/selectors/cart/cart.selector.ts

```diff
@@ -10,11 +10,21 @@ import { daffComparePersonalAddresses } from '@daffodil/geography';
 
 import { getDaffCartFeatureSelector } from '../cart-feature.selector';
 import { DaffCart } from '../../models/cart';
-import { DaffCartReducerState, DaffCartReducersState, DaffCartOperationType } from '../../reducers/public_api';
+import { DaffCartReducerState, DaffCartReducersState, DaffCartOperationType, DaffCartLoading } from '../../reducers/public_api';
 import { DaffCartOrderResult } from '../../models/cart-order-result';
 import { DaffCartItem } from '../../models/cart-item';
 import { DaffCartTotalTypeEnum, DaffCartTotal } from '../../models/cart-total';
 
+/**
+ * Util function to project loading selectors
+ */
+function projectAllLoadingSelectors(
```

Comment:
was this going to be for the selectors below, then you ended up not using it?

Reply:
put another way, I'm just not sure what this is for

Reply:
whoops yeah it was

Reply:
removed